src/routes/boot.ts

```diff
@@ -177,6 +179,24 @@ const getSquads = async (
     });
   });
 
+const getFeeds = async ({
+  con,
+  userId,
+}: {
+  con: DataSource;
+  userId: string;
+}): Promise<Feed[]> => {
+  const feeds = con.getRepository(Feed).find({
+    where: {
+      id: Not(userId),
+      userId,
+    },
+    take: maxFeedsPerUser,
+  });
+
+  return feeds;
+};
```

Comment:
Should we store a cache in redis?

Reply:
nah, it should be fast enough, if needed we can add down the line

Reply:
I don't want to complicate with invalidation at this point

Reply:
👍